bin/serverless.js

```diff
@@ -8,25 +8,29 @@ const logError = require('../lib/classes/Error').logError;
 const uuid = require('uuid');
 const initializeErrorReporter = require('../lib/utils/sentry').initializeErrorReporter;
 
-Error.stackTraceLimit = Infinity;
+(() => {
+  if (Number(process.version.split('.')[0].slice(1)) >= 8) {
+    const componentsCli = require('@serverless/cli');
+    if (componentsCli.runningComponents()) {
+      return componentsCli.runComponents().then(() => process.exit(0));
```

Comment:
Why `process.exit()` shortcut? 

We should rather let process to gracefully end, otherwise it may hide some issues (e.g. unhandled rejections).

We already have problems due to such handling in SLS, and plan to revert from that.

Reply:
makes sense. Will revert it.